test/mock-results.js

```diff
@@ -21,6 +21,7 @@ module.exports = function(stats, options) {
               timestamp: "1970-01-01T00:00:00", // ISO timestamp truncated to the second
               tests: "0",
               failures: "0",
+              file: "spec.js",
```

Comment:
note that the existing behavior passes through a `file` attribute for the root suite as is, but the mock result data didn't reflect it because the test setup doesn't set the `file` property on the root `Mocha.Suite` object